이정인/0901.py

```diff
@@ -0,0 +1,16 @@
+import collections
```

Comment:
컬렉션 패키지의 deque 모듈만 사용하니까 deque 모듈만 불러오는 게 어떨까요~?
```suggestion
from collections import deque
```

Reply:
저렇게 하는 게 더 효율적이겠네요 !